src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ManageContent/AddCommentRequest.cs

```diff
@@ -5,11 +5,12 @@ namespace GovUk.Education.ExploreEducationStatistics.Admin.Controllers.Api.Manag
 {
     public class AddCommentRequest
     {
+        public Guid UserId { get; set; }
```

Comment:
Unless you are able to create a comment on behalf of someone else I don't think this should be in the request. The user can be determined from the HTTP context making the request.

This implementation allows the user to be altered.

Reply:
When comments are created you can use UserService to obtain UserId.

Reply:
I'd agree, we shouldn't be taking this from the request